modules/sentinel-data-connector-aws-cloudtrail/vars.tf

```diff
@@ -0,0 +1,27 @@
+variable "aws_role_arn" {
+  description = "The ARN of the AWS CloudTrail role, which is connected to this AWS CloudTrail Data Connector."
+  type        = string
+}
+
+variable "log_analytics_workspace_id" {
+  description = "The ID of the Log Analytics Workspace that this AWS CloudTrail Data Connector resides in."
+  type        = string
+}
+
+variable "name" {
+  description = "The name which should be used for this AWS CloudTrail Data Connector."
+  type        = string
+}
+
+variable "timeouts" {
+  description = "nested block: NestingSingle, min items: 0, max items: 0"
+  type = set(object(
+    {
+      create = string
+      delete = string
+      read   = string
+      update = string
+    }
+  ))
+  default = []
+}
```

Comment:
As a user I would probably not understand what this is when reading the description.

Can we add some additional words to describe what this is and what value(s) it could take?

https://www.terraform.io/language/resources/syntax#operation-timeouts

I'm also confused by this being a set as to how multiple [timeouts](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/sentinel_data_connector_azure_security_center#timeouts) blocks would be handled.  Should this just be an object and default to 30m, 5m, 30m?

Reply:
I think it should just be an object, let me confirm that
Also, passing `default = []` will automatically make it use the [timeout values specified in the provider](https://github.com/hashicorp/terraform-provider-azurerm/blob/8a11ed64abb6403dfdfabbb16d7f68fafd671a3f/internal/services/sentinel/sentinel_data_connector_aws_cloud_trail.go#L32)
Will add a better description too

Reply:
Or can i just remove the timeouts parameter altogether? So that the default values are used everywhere.
Can go with this approach if the need to customize timeouts are pretty low

Reply:
As per our conversation you are going to remove these.  👍